Bomb.py

```diff
@@ -0,0 +1,40 @@
+import pygame
+from constants import *
+
+class Bomb:
+    def __init__(self, x, y, power):
+        self.x = x
+        self.y = y
+        self.power = power
+        self.timer = 0
+
+    def __del__(self):
+        pass
+
```

Comment:
Will you need it in a future or it could be deleted?

Reply:
Turns out it's not needed so I deleted it.